### PR TITLE
Adding keymap for KBDFans Kbd67 MKII RGB

### PR DESCRIPTION
**Pull request by xplusplus**
## Description
Adding a keymap for the KBDFans Kbd67 MKII RGB

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by drashna** [comment]
Looks good.  

THe only issue I can see, is that we generally like to try to keep the keymap folder name the same as the github account name, as this makes it easier to keep track of ownership of the keymaps.

File: keyboards/kbdfans/kbd67/mkiirgb/keymaps/codecoffeecode/keymap.c [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+   * | Shift      |  Z  |  X  |  C  |  V  |  B  |  N  |  M  |  ,  |  .  |  /  |   MO(2)    | Up  | PgDn |
+   * +-------------------------------------------------------------------------┬---┬-------------+------+
+   * | LCtrl | LGUI | LAlt |               Space                 | Btn2 | MO(1)|   | Left  |Down | Right|
+   * `-------------------------------------------------------------------------┘   └-------------+------´
+   */
+    [0] = LAYOUT_65_ansi_blocker( /* Base */
+        KC_GESC,       KC_1,     KC_2,     KC_3,  KC_4,  KC_5,  KC_6,    KC_7,  KC_8,    KC_9,     KC_0,     KC_MINS,  KC_EQL,  KC_BSPC,   KC_HOME,\
+        KC_TAB,        KC_Q,     KC_W,     KC_E,  KC_R,  KC_T,  KC_Y,    KC_U,  KC_I,    KC_O,     KC_P,     KC_LBRC,  KC_RBRC, KC_BSLASH, KC_END, \
+        CTL_T(KC_CAPS),KC_A,     KC_S,     KC_D,  KC_F,  KC_G,  KC_H,    KC_J,  KC_K,    KC_L,     KC_SCLN,  KC_QUOT,           KC_ENT,    KC_PGUP,\
+        KC_LSFT,       KC_Z,     KC_X,     KC_C,  KC_V,  KC_B,  KC_N,    KC_M,  KC_COMM, KC_DOT,   KC_SLSH,  MO(2),             KC_UP,     KC_PGDN,\
```
Comment (fauxpark) [inline keyboards/kbdfans/kbd67/mkiirgb/keymaps/codecoffeecode/keymap.c]: The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater without them.

File: keyboards/kbdfans/kbd67/mkiirgb/keymaps/codecoffeecode/keymap.c [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+void matrix_scan_user(void)
+{
+  //user matrix
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record)
+{
+	  return true;
+}
```
Comment (fauxpark) [inline keyboards/kbdfans/kbd67/mkiirgb/keymaps/codecoffeecode/keymap.c]: These functions can be deleted.

**Comment by xplusplus** [comment]
> 
> 
> Looks good.
> 
> THe only issue I can see, is that we generally like to try to keep the keymap folder name the same as the github account name, as this makes it easier to keep track of ownership of the keymaps.

@drashna I agree. Long ago I attempted to change my github account name, before name changes were actually allowed. I'll make sure to actually change it before submitting another keymap PR.

**Pull request merged**